app/processors/message_processor.py

```diff
@@ -24,9 +25,17 @@ def process(self, msg, tx_id):
         )
 
         document = get_doc_from_store(tx_id)
-        cs_processor = CommonSoftwareProcessor(self.logger, document, self._ftp)
-
-        cs_processor.process()
-        cs_processor.logger.info("Processed successfully",
-                                 tx_id=cs_processor.tx_id,
-                                 )
+        if document['survey_id'] == '144':
```

Comment:
Might be better to use a constant or list of constants so that it's easy to add more `CORA` destined surveys to this.